cafebabel/templates/base.html

```diff
@@ -56,12 +56,15 @@
   </ul>
 </nav>
 
+<button id=participate><a href=#>Participate</a></button>
```

Comment:
Interactive element (`a`) inside of an interactive element (`button`) looks confusing.
It should either be a link to a different content (`a`), or be an interactive element (`button`).

`a href=#` makes me think it's not an actual link…